chrome/content/zotero/xpcom/style.js

```diff
@@ -44,11 +44,19 @@ Zotero.Styles = new function() {
 	this.ns = {
 		"csl":"http://purl.org/net/xbiblio/csl"
 	};
-
-	// TEMP
-	// Until we get asynchronous style loading, load renamed styles at startup, since the
-	// synchronous call we were using breaks the first drag of the session (on OS X, at least)
+	
 	this.preinit = function () {
+		// Upgrade style locale prefs for 4.0.27
```

Comment:
Yes, but it's not a one-time thing, since people might still be upgrading old Zotero installations in the future. So this code is necessary to deal with installations before 4.0.27, and desired/necessary for any version beyond 4.0.26. That's a bit unclear now.